src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/AttributesTests.cs

```diff
@@ -320,5 +320,11 @@ public static void InterpolatedStringHandlerArgumentAttributeTests(string firstP
             Assert.Equal(firstParameterName, attr2.Arguments[0]);
             Assert.Equal("param2", attr2.Arguments[1]);
         }
+
+        [Fact]
+        public static void RequiredMemberAttributeTests()
+        {
+            new RequiredMemberAttribute();
+        }
```

Comment:
The two test files will need to be added to https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj as well.

Reply:
This one already is, but I'll add the other.